### DISCUSSION TITLE
[version：1.2.0-beta.5]Element-Plus  el-drop-item divided. its style has a problem?

### DISCUSSION THREAD

**Discussion by 2681756497** [element-plus/element-plus#4773] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/77483644/145558924-d3723a77-0aaf-42b7-bd61-2b332e8f0e94.png)

**Comment by emojiiii** [comment, marked as answer]
fix in #4638
Please upgrade `element-plus` to the latest version
